sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/ResourceExtensions.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using OpenTelemetry.Resources;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics;
+
+internal static class ResourceExtensions
```

Comment:
nit: fine for POC, but i wonder if we could refactor this class in the Exporter library and import it here.

Reply:
Getting PoC, should be our priority. 
This is a major refactor, better to have a separate copy for live metrics.